test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -472,3 +494,13 @@ func waitForReplicas(deploymentName, namespace string, cs clientset.Interface, t
 		framework.Failf("Timeout waiting %v for %v replicas", timeout, desiredReplicas)
 	}
 }
+
+// Equivalent of featuregatetesting.SetFeatureGateDuringTest
```

Comment:
this really doesn't seem correct... changing feature gate state in the e2e binary is not something I expect us to do... can you follow up to drop this

Reply:
This was removed as a part of https://github.com/kubernetes/kubernetes/pull/112444.